### PR TITLE
adding push notification accent color

### PR DESCRIPTION
**Pull request by peterbrown0**

### CODE REVIEW

File: CountlyConfig.js [hunk 1]
```diff
@@ -144,6 +145,39 @@ class CountlyConfig {
...
+        return this;
+    }
+
+    /**
+    * Method to set the push channel name and description
+    *
+    * @param {String} name channel name
+    * @param {String} description channel description
+    */
+    setPushNotificationChannel(name, description) {
```
Comment (ArtursKadikis) [inline CountlyConfig.js:164]: setPushNotificationChannel -> setPushNotificationChannelInformation

File: android/src/main/java/ly/count/android/sdk/react/CountlyReactNative.java [hunk 2]
```diff
@@ -297,6 +300,20 @@ private void populateConfig(JSONObject _config) {
         }
     }
 
+    private void setHexNotificationAccentColor(final String hex) {
```
Comment (ArtursKadikis) [inline android/src/main/java/ly/count/android/sdk/react/CountlyReactNative.java:303]: let's do some value validation

File: android/src/main/java/ly/count/android/sdk/react/CountlyReactNative.java [hunk 3]
```diff
@@ -297,6 +300,20 @@ private void populateConfig(JSONObject _config) {
...
+        try {
+            int color = Color.parseColor(hex);
+            CountlyPush.setNotificationAccentColor(
+                Color.alpha(color),
+                Color.red(color),
+                Color.green(color),
+                Color.blue(color)
+            );
+        } catch (IllegalArgumentException e) {
+            log("setHexNotificationAccentColor: invalid HEX color value: " + hex + " Error: " + e, LogLevel.DEBUG);
```
Comment (ArtursKadikis) [inline android/src/main/java/ly/count/android/sdk/react/CountlyReactNative.java:313]: this should be error

File: CountlyConfig.js [hunk 4]
```diff
@@ -132,6 +132,7 @@ class CountlyConfig {
 
     /**
     * Method to set the push token type
+    * @deprecated
```
[the remark this answers is not in the document]
Comment (peterbrown0) [reply CountlyConfig.js:135]: done

**Pull request closed without merging**